Description: partially revert changes in https://chromium.googlesource.com/chromium/src/+/40a8d84 that require a version of llvm-ar that's more recent than what is in xenial
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/build/toolchain/gcc_toolchain.gni
+++ b/build/toolchain/gcc_toolchain.gni
@@ -337,7 +337,7 @@ template("gcc_toolchain") {
 
       # TODO(thomasanderson): Remove the special ChromeOS case once
       # https://crbug.com/829956 is fixed.
-      if (default_toolchain == "//build/toolchain/cros:target") {
+      if (true) {
         extra_arflags = "rcsD"
       } else if (current_os == "aix") {
         # We use slightly different arflags for AIX.
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1560,7 +1560,7 @@ config("thin_archive") {
   if ((is_posix && !is_nacl) || is_fuchsia) {
     # TODO(thomasanderson): Enable on ChromeOS builds once
     # https://crbug.com/829956 is fixed.
-    if (default_toolchain != "//build/toolchain/cros:target") {
+    if (false) {
       arflags = [ "-T" ]
     }
   }
